Content.Shared/Humanoid/Markings/Marking.cs

```diff
@@ -12,7 +12,7 @@ public sealed class Marking : IEquatable<Marking>, IComparable<Marking>, ICompar
         [DataField("markingColor")]
         private List<Color> _markingColors = new();
 
-        private Marking()
+        public Marking()
         {
         }
```

Comment:
This should stay private, there was an engine PR recently that should make yaml serialization look for private constructors

Reply:
That's good to know. Though I don't understand why should the constructor remain private

Reply:
several parts of the Marking class are read only by design, making the empty constructor useless if public

Reply:
> several parts of the Marking class are read only by design, making the empty constructor useless if public

Not quite, I was forced to make this change as the serializer would not work explicitly because there was no empty constructor. As soon as changing the accessability to public it worked as should be obvious. I understand that recent changes may have or are to be fixed getting parameterless constructor from a class that is made private

Reply:
it would be useless in terms of using it in other code - while you can make an empty marking, it has no purpose other than being serializable (and electro said that private zero-parameter constructors are now valid for serialization as well) so this review still stands